src/css/base.scss

```diff
@@ -1746,6 +1745,10 @@ button.anchor {
     max-width: 300px;
 }
 
+.dc__mxw-400 {
```

Comment:
A minor one but please maintain the order of digits toward increasing order always

Reply:
Done